### DISCUSSION TITLE
HTML only Project

### DISCUSSION THREAD

**Discussion by andersontz** [shadcn-ui/ui#3492] [General]
Hello, I'm interested in using Shadcn in a web, html project. How to start? Which path should I follow?

**Comment by ruru-m07** [comment]
with only html is it not possible because shadcn ui is component base you can use react but not only html

**Reply by andersontz** [reply]
And can I start working on an HTML version or will I violate the license?
Can I create a derivation with another name and just base it on the visual interface?

**Reply by ruru-m07** [reply]
i think it's against to the license but if you create fully your own component so it's fine but still it's not possible to use shadcn ui component with only html because they use [https://www.radix-ui.com](https://www.radix-ui.com) so u need to install this package to use shadcn-ui but you can't use with plain html

id you start to create your own component library than i always here to help you
